compiler/passes/convert-uast.cpp

```diff
@@ -4366,12 +4367,16 @@ struct Converter {
 };
 
 bool LoopAttributeInfo::insertGpuEligibilityAssertion(BlockStmt* body) {
+  bool inserted = false;
   if (assertOnGpuAttr) {
-    body->insertAtTail(new CallExpr(PRIM_ASSERT_ON_GPU,
```

Comment:
Have you removed this primitive? I don't think you did.

Reply:
I didn't, and don't intend to in this PR, though my goal in the long term is to remove it.